rtic-monotonics/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ For each category, *Added*, *Changed*, *Fixed* add new entries at the top!
 
 - Updated esp32c3 dependency to v0.28.0
 - Updated esp32c3 dependency to v0.27.0
+- Panic if STM32 prescaler value would overflow
```

Comment:
This changelog entry does not belong to this release. 

You should create a new "changed" header under "unreleased" header and place it there instead :)

Reply:
ah, I did it too quickly :sweat_smile: 

should be fixed now